src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -357,6 +358,12 @@ private static void syncConfig(boolean load)
         prop.setLanguageKey("forge.configgui.forgeLightPipelineEnabled");
         propOrder.add(prop.getName());
 
+        prop = config.get(Configuration.CATEGORY_CLIENT, "selectiveResourceReloadEnabled", true,
+                "When enabled, makes specific reload tasks such as language changing quicker to run.");
```

Comment:
This needs major testing with current known modpacks. So we can decide if it should be opt-in or opt-out. I'm in favor of opt-in due to how much this has the potential to break.